examples/basic-example/docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
       # - DEBUG_MODE=true
     ports:
       - 8000:80
+    expose:
```

Comment:
We use expose only for this compose ?

Reply:
`expose` is only for the sake of documentation. I'm not sure if it is more useful than confusing for a Docker user 🤔